spec/factorybot_lint_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# spec/lint_spec.rb
+
+require 'rails_helper'
+
+RSpec.describe 'Lint' do
+  it 'FactoryBot factories' do
+    conn = ActiveRecord::Base.connection
```

Comment:
This is not necessary, transactions are already [automatically](https://relishapp.com/rspec/rspec-rails/docs/transactions) [rolled back](https://github.com/turniere/turniere-backend/blob/master/spec/rails_helper.rb#L44).